src/components/ebay-menu/test/test.browser.js

```diff
@@ -11,6 +11,29 @@ afterEach(cleanup);
 /** @type import("@marko/testing-library").RenderResult */
 let component;
 
+describe('typeahead functionality', () => {
```

Comment:
Do we want a test to check the timeout as well, like:
Type a
Wait for 2 seconds
Type l
Should focus on item starting with l

Or is this covered in the makeup portion?

Also do we want to check the functionality if nothing matches? Like if someone types in "a", "l" "x" that nothing is focused or something, or the last one is still focused?

Reply:
I think the nothing matches would be a good test case.

Reply:
done